package.json

```diff
@@ -45,6 +46,7 @@
     "vite": "^2.2.3"
   },
   "dependencies": {
+    "dom-parser": "^0.1.6",
```

Comment:
May I ask how you found this module? It's not very popular and the last version is 2 years old, but it seems to be stable. Feel free to use it 👍